### ISSUE TITLE
Game crashes on putting worktable tools in

### ISSUE THREAD

**Issue by JWTHDYTWA** [codetaylor/artisan-worktables#41]
Minecraft crashes when i put tools inside the worktables after update which added liquids usage.

It happens on modified FTB Revelation modpack.

Here is summary error from launcher:
The game crashed whilst updating screen events
Error: java.lang.NullPointerException: Updating screen events

Crash log:
https://gist.github.com/Warrock3280/23c5ab18f0de1e601224a52a0b197445#file-crash-2018-02-01_07-38-30-client-txt

**Comment by JWTHDYTWA** [comment]
In addition, no recipes are displayed via JEI. Here is used script if needed:
https://gist.github.com/Warrock3280/879fe05a80123e764ee57a60f6b4e470#file-03_artisan-zs

**Comment by MasterEnderman** [comment]
I'm having the same Problem.

Crash Report: https://pastebin.com/FGtfcf1J
Forge: 14.23.1.2606
ArtisianWortables: 1.10.17
Athenaeum: 1.6.3

Script: https://pastebin.com/s13Ud8wp

**Comment by MasterEnderman** [comment]
@warrock3280 downgrading fixes the issue for now

**Comment by codetaylor** [comment]
Reproduced with:
```
import mods.artisanworktables.Worktable;

Worktable.createRecipeBuilder("carpenter")
    .setShaped([
        [<minecraft:planks>],
        [<minecraft:planks>],
        [<minecraft:planks>]])
    .setTool(<ore:carpenters_hammer>, 3)
    .addOutput(<minecraft:gravel>)
    .create();
```

Truncated Log:
```
java.lang.NullPointerException: null
	at com.codetaylor.mc.artisanworktables.modules.worktables.recipe.RecipeWorktable.getFluidIngredient(RecipeWorktable.java:147) ~[RecipeWorktable.class:?]
	at com.codetaylor.mc.artisanworktables.modules.worktables.integration.jei.JEIRecipeWrapperWorktable.getIngredients(JEIRecipeWrapperWorktable.java:130) ~[JEIRecipeWrapperWorktable.class:?]
	at mezz.jei.recipes.RecipeRegistry.getIngredients(RecipeRegistry.java:279) ~[RecipeRegistry.class:?]
	at mezz.jei.recipes.RecipeRegistry.addRecipeUnchecked(RecipeRegistry.java:267) ~[RecipeRegistry.class:?]
[...]
```

Fails with any recipe that doesn't define a fluid.

**Comment by codetaylor** [comment]
Fix available in version 1.12.2-1.10.18

Thanks for the succinct, clean bug reports!
